inference-engine/src/mkldnn_plugin/mkldnn_infer_request.cpp

```diff
@@ -94,44 +91,44 @@ void MKLDNNPlugin::MKLDNNInferRequest::InferImpl() {
             InferenceEngine::TBlob<float> *in_f = nullptr;
             switch (input.second->getTensorDesc().getPrecision()) {
```

Comment:
To me, the logical evolution of the refactoring would be moving (or even removing!) this huge (and straighforward) switch as well

Reply:
I agree. @mandrono please think about the solution